frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -143,11 +143,19 @@ export interface ServerlessScaling {
 }
 
 export enum GitTypes {
-  '' = 'Please choose Git type',
-  github = 'GitHub',
-  gitlab = 'GitLab',
-  bitbucket = 'Bitbucket',
-}
+  github = 'github',
+  gitlab = 'gitlab',
+  bitbucket = 'bitbucket',
+  unsure = 'other',
+  invalid = '',
+}
+
+export const GitReadableTypes = {
+  [GitTypes.github]: 'GitHub',
+  [GitTypes.gitlab]: 'GitLab',
+  [GitTypes.bitbucket]: 'Bitbucket',
+  [GitTypes.unsure]: 'Other',
```

Comment:
There were a lot of magic strings in play that I had to track down. I choose this was a good opportunity to unify them as an enum with a map of readable values.